test/Conversion/ExportVerilog/sv-dialect.mlir

```diff
@@ -343,14 +343,13 @@ hw.module @M1<param1: i42>(%clock : i1, %cond : i1, %val : i8) {
 
   // CHECK-NEXT: `ifdef FOO
   sv.ifdef "FOO" {
-    // CHECK-NEXT: wire {{.+}} = "THING";
     %c1 = sv.verbatim.expr "\"THING\"" : () -> i1
 
     // CHECK-NEXT: initial begin
     sv.initial {
-      // CHECK-NEXT: fwrite(32'h80000002, "%d", {{.+}});
+      // CHECK-NEXT: fwrite(32'h80000002, "%d", "THING");
       sv.fwrite "%d" (%c1) : i1
-      // CHECK-NEXT: fwrite(32'h80000002, "%d", {{.+}});
+      // CHECK-NEXT: fwrite(32'h80000002, "%d", "THING");
```

Comment:
Can you add a flavor of this where the expression is really ling and should be spilled to ensure it is?

Reply:
It seems currently the length of verbatim string is not much consider when the mission:
```mlir
hw.module @Foo() { 
  sv.initial {
  %c2 = sv.verbatim.expr "\"VERY_LONG_LINE_VERY_LONG_LINE_VERY_LONG_LINE_VERY_LONG_VERY_LONG_LINE_VERY_LONG_LINE_VERY_LONG_LINE_VERY_LONG\"" : () -> i1
    sv.fwrite "%d" (%c2) : i1
  }
}
```
The output is:
```sv
module Foo();   // foo.mlir:1:1
  initial       // foo.mlir:2:3
    $fwrite(32'h80000002, "%d", "VERY_LONG_LINE_VERY_LONG_LINE_VERY_LONG_LINE_VERY_LONG_VERY_LONG_LINE_VERY_LONG_LINE_VERY_LONG_LINE_VERY_LONG");       // foo.mlir:3:9, :4:5
endmodule
```

I'll submit another PR.